PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/webapp/webappconfig/RiderWebAppRunState.kt

```diff
@@ -0,0 +1,441 @@
+import okhttp3.Response
+import java.io.*
+import java.util.zip.ZipOutputStream
+
+class RiderWebAppRunState(project: Project,
+                          private val myModel: AzureDotNetWebAppSettingModel) : AzureRunProfileState<WebApp>(project) {
+
+    companion object {
+        private const val TARGET_NAME = "WebApp"
+
+        // Subscription
+        private const val SUBSCRIPTION_ID_NOT_DEFIED = "Subscription ID is not defined"
+
+        private const val PROJECT_NOT_DEFINED = "Project is not defined"
+
+        private const val WEB_APP_START = "Start Web App '%s'..."
+        private const val WEB_APP_STOP = "Stop Web App '%s'..."
+        private const val WEB_APP_CREATE = "Creating Web App '%s'..."
+        private const val WEB_APP_CREATE_SUCCESSFUL = "Web App  is created successfully, id '%s'"
+        private const val WEB_APP_GET_EXISTING = "Get existing Web App with Id '%s'"
+        private const val WEB_APP_ID_NOT_DEFINED = "Web App ID is not defined"
+        private const val WEB_APP_NAME_NOT_DEFINED = "Web App Name is not defined"
+
+        private const val RESOURCE_GROUP_NAME_NOT_DEFINED = "Resource Group Name is not defined"
+
+        private const val APP_SERVICE_PLAN_ID_NOT_DEFINED = "App Service Plan ID is not defined"
+        private const val APP_SERVICE_PLAN_NAME_NOT_DEFINED = "App Service Plan Name is not defined"
+        private const val APP_SERVICE_PLAN_REGION_NOT_DEFINED = "App Service Plan Region is not defined"
+        private const val APP_SERVICE_PLAN_PRICING_TIER_NOT_DEFINED = "App Service Plan Pricing Tier is not defined"
+
+        private const val DEPLOY_SUCCESSFUL = "Deploy successfully!"
+        private const val DEPLOY_FAILED = "Deploy failed"
+        private const val DEPLOY_GET_CREDENTIAL = "Getting deployment credential..."
+
+        private const val PROJECT_ARTIFACTS_COLLECTING = "Collecting '%s' project artifacts..."
+        private const val PROJECT_ARTIFACTS_COLLECTING_FAILED = "Failed collecting project artifacts. Please see Build output"
+
+        private const val ZIP_FILE_CREATE_FOR_PROJECT = "Creating '%s' project ZIP..."
+        private const val ZIP_FILE_CREATE_SUCCESSFUL = "Project ZIP is created: '%s'"
+        private const val ZIP_FILE_NOT_CREATED = "Unable to create a ZIP file"
+        private const val ZIP_FILE_DELETING = "Deleting ZIP file '%s'"
+        private const val ZIP_DEPLOY_START_PUBLISHING = "Publishing ZIP file..."
+        private const val ZIP_DEPLOY_PUBLISH_SUCCESS = "Published ZIP file successfully"
+        private const val ZIP_DEPLOY_PUBLISH_FAIL = "Fail publishing ZIP file"
+
+        private const val URL_AZURE_BASE = ".azurewebsites.net"
+        private const val URL_KUDU_BASE = ".scm$URL_AZURE_BASE"
+        private const val URL_KUDU_ZIP_DEPLOY = "$URL_KUDU_BASE/api/zipdeploy"
+
+        private const val COLLECT_ARTIFACTS_TIMEOUT = 180000L
+        private const val DEPLOY_TIMEOUT = 180000L
```

Comment:
it's good to show what unit of measure was used in naming, like in SLEE_TIME_MS

Reply:
Done.